env-defaults

```diff
@@ -95,6 +95,9 @@ DEFAULT_RESULT_PAGE_SIZE=2000
 # Default number of jobs listed in a page
 DEFAULT_JOB_LIST_PAGE_SIZE=10
 
+# TODO change this to a smaller number when aggregating services are updated to handle paged catalogs
```

Comment:
Make sure we write a ticket(s) to update the aggregating services and put the ticket number(s) here.

Reply:
done